ci/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set version = "2.0.1" %}
+
+package:
+  name: pandas
+  version: {{ version }}
+
+source:
+  url: https://github.com/pandas-dev/pandas/releases/download/v{{ version }}/pandas-{{ version }}.tar.gz
+  sha256: 19b8e5270da32b41ebf12f0e7165efa7024492e9513fb46fb631c5022ae5709d
```

Comment:
I guess we may want to build the package with the lastest pandas from git, not the latest version? Not sure if this will fail when new dependencies are added or similar things, and Dask seems to do that:

https://github.com/dask/dask/blob/main/continuous_integration/recipe/meta.yaml#L11

Reply:
Ah I see. Good call I'll change this